### ISSUE TITLE
Unsupported Ellipsoids?

### ISSUE THREAD

**Issue by harumscarum** [JeremyBYU/bpyproj#22]
I have noticed this issue long time ago, but just recently was able to isolate it - apparently bpyproj do not support projections where are datum/spheroid different from WGS 84 which cause slight scale/rotate transformations - really noticeable in big scenes (up to 10 meters for 200 km scene)

here is example
- Specific projection for the Kaliningrad https://epsg.io/20004 (DATUM"Pulkovo_1995", SPHEROID "Krassowsky 1940") and I used both EPSG code and Proj4 Params, but result do not match GIS data from QGIS or GlobalMapper
- However, if WGS 84 projection used https://epsg.io/32634 - result delivered with bpyproj match other GIS sources perfectly

Is there any way to fix this?
And do I understand correctly that bpyroj do not use online database for the most actual EPSG data, but use Proj4 strings stored locally \scripts\addons\bpyproj\dependencies\binaries\windows_64_37\site-packages\pyproj\data\
May I suggest you to use WKT to get accurate result? https://inbo.github.io/tutorials/tutorials/spatial_crs_coding/

UPD 
---
I'm checking projections and just found one more datum/ellipsoid which deliver accurate results - DATUM"NAD83(NSRS2007)", SPHEROID "GRS 1980" - hope this helps

**Comment by harumscarum** [comment]
Would appreciate any help with the Hong Kong 1980 Grid System - https://epsg.io/2326
Result delivered with the bpyproj do not match other GIS sources no matter what I use EPSG or Proj4 string

**Comment by vvoovv** [comment]
> Is there any way to fix this? And do I understand correctly that bpyroj do not use online database for the most actual EPSG data, but use Proj4 strings stored locally \scripts\addons\bpyproj\dependencies\binaries\windows_64_37\site-packages\pyproj\data May I suggest you to use WKT to get accurate result? https://inbo.github.io/tutorials/tutorials/spatial_crs_coding/

Bpyproj does not use any online database and does not initiate any internet connection to get data.

**Comment by harumscarum** [comment]
Yes, I understand that bpyproj stores all projections locally in the folder \scripts\addons\bpyproj\dependencies\binaries\windows_64_37\site-packages\pyproj\data.

The issue arises when certain projections with non-standard datum/spheroid are used. In such cases, Blender generates geometry that significantly deviates from accurate GIS data (up to 20 meters over 100 km). These deviations occur in both scale and distortion.

Examples of unsupported datum/spheroid include:
DATUM "Qatar National Datum 1995", ELLIPSOID "International 1924"
DATUM "Pulkovo_1995", SPHEROID "Krassowsky 1940"
and others.

I'm looking for a way to improve bpyproj so that it works correctly with all projections. This is a commercial offer, as I'm ready to pay for the development.
